test/classes/delete-free/type-casting.good

```diff
@@ -0,0 +1,6 @@
+type-casting.chpl:12: In function 'makeBorrowedBang1':
```

Comment:
My intention with this test is that the regular .good file not have any errors in it and any error cases end up in type-casting.errors.good. That means that the `check` calls at line 54 and 60 need to move in to the conditional `if DuplicateManagement` on their next lines.